cluster-autoscaler/core/utils.go

```
@@ -152,6 +154,20 @@ func GetNodeInfosForGroups(nodes []*apiv1.Node, cloudProvider cloudprovider.Clou
 	return result, nil
 }
 
+func sanitizeTemplateNodeLables(node *apiv1.Node) *apiv1.Node {
+	obj, _ := api.Scheme.DeepCopy(node)
```

Why ignore error value?